calblink.go

```diff
@@ -944,6 +944,14 @@ func main() {
 	if *debugFlag {
 		debugOut = os.Stdout
 	}
+	clientSecretPath := "/path/to/client_secret.json"
```

Comment:
Please use the value of clientSecretFlag instead of a hard-coded path.

Reply:
Yes Done changes